LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Eddie Jaoude Community
+Copyright (c) 2020-2021 EddieHub
 
```

Comment:
Why `2020-2021`? I think `2021` is better.

Reply:
The project was created in 2020 (I guess), and it still continues in 2021. 🙂

Reply:
It still continues in 2021, that's why `2020-2021`.

Reply:
Aa, No problem but its needed to edit in all years.